crates/scsi/rustfmt.toml

```diff
@@ -0,0 +1,4 @@
+imports_granularity = "Crate"
+group_imports = "StdExternalCrate"
+wrap_comments = true
+normalize_comments = true
```

Comment:
Do we need/want this?

Reply:
I think all the clippy and rustfmt rules are covered by the common CI stuff. If that is the case I don't think we need to make it explicit here as the rust-vmm coding style is documented for all crates.